src/urdf_loader.cpp

```diff
@@ -208,11 +208,22 @@ std::string resolveURI(const std::string &path)
         // Return the canonical path
         return file_path.string();
     } else {
-        RAVELOG_WARN("Cannot handle mesh URI type '%s'.\n");
+        RAVELOG_WARN("Cannot handle this type of URI.\n");
         return "";
     }
 }
 
+std::string resolveURIorPath(const std::string &path)
+{
+    using boost::algorithm::starts_with;
+
+    if (starts_with(path, "package://") || starts_with(path, "package://")) {
```

Comment:
Good catch. The second one should be file://. :-)

On Thu, Apr 9, 2015 at 3:25 PM, cdellin <notifications@github.com> wrote:

> In src/urdf_loader.cpp
> <https://github.com/personalrobotics/or_urdf/pull/12#discussion_r28094089>
> :
>
> >          return "";
> >      }
> >  }
> >
> > +std::string resolveURIorPath(const std::string &path)
> > +{
> > +    using boost::algorithm::starts_with;
> > +
> > +    if (starts_with(path, "package://") || starts_with(path, "package://")) {
>
> I think you can just call starts_with once at this point?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/personalrobotics/or_urdf/pull/12/files#r28094089>.
>